<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Civilité: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom et Prénom:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mail:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Languages de programmation:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch_autres">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>100</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>autres</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch_php">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>100</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>php</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch_js">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>100</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>js</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch_java">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>340</y>
      <width>100</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>java</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch_cpp">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>100</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>c++</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch_py">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>100</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>python</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>119</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>m.</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="mille">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>119</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>mlle.</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="mme">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>119</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>mme.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_enreg">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>380</y>
      <width>97</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enregistrer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_afficher">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher fichier</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tb_affich">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>401</width>
      <height>201</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>401</width>
      <height>16777215</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>Num</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Civilité</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>nom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>gcprog</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="inp_np">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inp_mail">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_langue">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>86</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>anglais</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>francais</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>allmend</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>espanoel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>arabe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>italien</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>americain</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>suedois</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>finlandois</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>burger</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>hanen</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_eff">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>640</y>
      <width>97</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Effacer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_quitter">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>640</y>
      <width>97</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 36px;</string>
    </property>
    <property name="text">
     <string>Form </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Premier Langue Maitrisee:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
